dapr/proto/operator/v1/operator.proto

```diff
@@ -34,6 +34,10 @@ service Operator {
   rpc ListResiliency (ListResiliencyRequest) returns (ListResiliencyResponse) {}
   // Returns a list of pub/sub subscriptions, ListSubscriptionsRequest to expose pod info
   rpc ListSubscriptionsV2 (ListSubscriptionsRequest) returns (ListSubscriptionsResponse) {}
+  // Returns a list of http endpoints
+  rpc ListHTTPEndpoints (ListHTTPEndpointsRequest) returns (ListHTTPEndpointsResponse) {}
+  // Sends events to Dapr sidecars upon http endpoint changes.
+  rpc HTTPEndpointsUpdate (HTTPEndpointsUpdateRequest) returns (stream HTTPEndpointsUpdateEvent) {}
```

Comment:
```suggestion
  rpc HTTPEndpointsSubscribe (HTTPEndpointsUpdateRequest) returns (stream HTTPEndpointsUpdateEvent) {}
```
Here we are not updating the endpoint resource. I am open to other names also but here we are subscribing to listen to updates.

Reply:
I went with Update as opposed to subscribe as the similar functionality for components changes is called `ComponentUpdate`. If we change one, then I think we should change all for consistency. Thoughts?

Reply:
As we're moving to support hot-reloading and continuously listening on resource updates, going with `Update` is the right choice.